includes/admin/settings/register-settings.php

```diff
@@ -937,17 +958,13 @@ function edd_get_registered_settings() {
 
 		}
 
-		$edd_settings['misc']['privacy'][] = array(
-			'id'   => 'payment_privacy_status_action_' . $status,
-			'name' => sprintf( _x( '%s Payments', 'payment status labels: Pending Payments', 'easy-digital-downloads' ), $label ),
-			'desc' => '',
-			'type'        => 'select',
-			'options'     => array(
-				'none'      => __( 'None', 'easy-digital-downloads' ),
-				'anonymize' => __( 'Anonymize', 'easy-digital-downloads' ),
-				'delete'    => __( 'Delete', 'easy-digital-downloads' ),
-			),
-			'std'         => $action,
+		$edd_settings['privacy']['export_erase'][] = array(
+			'id'      => 'payment_privacy_status_action_' . $status,
+			'name'    => sprintf( _x( '%s Payments', 'payment status labels: Pending Payments', 'easy-digital-downloads' ), $label ),
```

Comment:
I think if context is going to be provided here, it should point out where the text is used. Right now it's still random. Just noting that it's in the Export & Erase settings would help.